lib/trackler/track.rb

```diff
@@ -81,8 +81,11 @@ def test_pattern
       end
     end
 
-    def docs(image_path = DEFAULT_IMAGE_PATH)
-      OpenStruct.new(docs_by_topic(image_path))
+    def docs(positional_image_path_which_is_deprecated = nil, image_path: nil)
```

Comment:
Now try with:
`p bar || foo || 'DEFAULT'` instead of `p foo, bar`

3 possible inputs, 1 desired output.